static/js/chart/draw_bar.ts

```diff
@@ -317,7 +317,7 @@ export function drawStackBarChart(
     chartHeight,
     x,
     false,
-    labelToLink,
+    options?.disableEntityLink ? null : labelToLink,
```

Comment:
I'm not super familiar with this code but do we have tests for these functions / components? Would be good to add tests if we don't.

Reply:
We dont 😬  - testing web components doesnt work out of the box with our webdriver setup, and adding support has been on my todo list